contracts/mockContracts/MockKyberNetwork.sol

```diff
@@ -56,6 +56,46 @@ contract MockKyberNetwork {
         return destAmount;
     }
 
+    // @dev trade function with same prototype as KyberNetwork
```

Comment:
I don't understand why I see changes in this contract??

Reply:
This mock contract was lacking the `tradeWithHint` function. Added it in.